lib/wsdl.js

```diff
@@ -1558,7 +1558,11 @@ WSDL.prototype.objectToXML = function(obj, name, namespace, xmlns, first, xmlnsA
       }
 
       if (!Array.isArray(child)) {
-        parts.push(['<', ns, name, attr, xmlnsAttrib, '>'].join(''));
+        var insNull = '';
+        if(typeof child === 'object' && child === null) {
```

Comment:
I see, so we need this extra check to guarantee that this is really just `null`? Please don't get me wrong, I'm not criticising your code, I'm just curious about the code others write and I'm wondering why we need to check for `=== 'object'` too if we only want to catch `=== 'null'` for further processing? As far as I know there is no value which could give a truthy check on `typeof val === 'null'` and not for `typeof val === 'object'` at the same time?